### PR TITLE
Add news behaviors on map

### PR DESCRIPTION
**Pull request by enzoggqs**
## Motivation

- Show the path of the selected element
- Create a return button to navigate between the map
- Blur the header background

## Changes

- Created the path of the current element on header
- Blurred the header background
- Create a return button

## Status Checklist

- [x] Create prototype
- [x] Create return button
- [x] Create path to show current selected element path
- [x] Make the path clickable to navigate between layers

## Screenshots

Prototype: 
image

|Before|After|
|---|---|
|image|image|

## Testing

- Start the application
- Click in any state
- Click in any district
- Click on return button or in "Brasil" on path

### CODE REVIEW

File: src/components/Header/LayerRoute/LayerRoute.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const returnPath = () => {
+    if (selectedLayer === 'district') {
+      return (
+        <>
+          &nbsp;-&nbsp;
+          <div className="state" onClick={() => resetDistrictValues()}>
+            {selectedState?.properties.NM_UF}
+          </div>
+          &nbsp;-&nbsp;<div className="district">{selected?.properties.NM_MUN}</div>
+        </>
```
Comment (valdersonjr) [inline src/components/Header/LayerRoute/LayerRoute.tsx:37]: Idk if it was intentional this div being on the same line.

File: src/components/Header/styles.ts [hunk 2]
```diff
@@ -63,3 +69,38 @@ export const HeaderRightSide = styled.div`
...
+export const ReturnRouteButton = styled.div`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  background-color: white;
+  margin-right: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```
Comment (valdersonjr) [inline src/components/Header/styles.ts:106]: I think this might me duplicated.

**Review by valdersonjr — COMMENTED** [review]
Wow! You did a really greate job. I was impressed.

File: src/components/Header/LayerRoute/LayerRoute.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      setSelectedLayer('country');
+    }
+  }, [selected, selectedState]);
+
+  const returnPath = () => {
+    if (selectedLayer === 'district') {
+      return (
+        <>
+          &nbsp;-&nbsp;
+          <div className="place state" onClick={() => resetDistrictValues()}>
```
Comment (LeoSilvaGomes) [inline src/components/Header/LayerRoute/LayerRoute.tsx:33]: Did you try to create a button from mui?

File: src/components/Header/LayerRoute/LayerRoute.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    } else {
+      setSelectedLayer('country');
+    }
+  }, [selected, selectedState]);
+
+  const returnPath = () => {
+    if (selectedLayer === 'district') {
+      return (
+        <>
+          &nbsp;-&nbsp;
```
Comment (LeoSilvaGomes) [inline src/components/Header/LayerRoute/LayerRoute.tsx:32]: What do you think to use a tab and style it?

File: src/components/Header/LayerRoute/styles.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'district' ? 'bold' : 'normal')};
+  }
+
+  .place {
+    padding: 5px;
+    border-radius: 5px;
+    &:hover {
+      background-color: rgb(239, 239, 239, 0.3);
+      /* background-color: rgb(239, 239, 239); */
+      /* opacity: 0.1; */
```
Comment (LeoSilvaGomes) [inline src/components/Header/LayerRoute/styles.ts:36]: 🔪

File: src/components/Header/styles.ts [hunk 6]
```diff
@@ -21,6 +25,7 @@ export const HeaderContainer = styled.header<HeaderProps>`
   padding: 15px 20px;
 
   background: ${({ comparisonMode }) => (comparisonMode ? '#ffffffe6' : null)};
+  backdrop-filter: blur(2px);
```
Comment (LeoSilvaGomes) [inline src/components/Header/styles.ts:28]: ❤️

File: src/components/Header/styles.ts (hunk 2 above)
Comment (LeoSilvaGomes) [reply src/components/Header/styles.ts:106]: @enzoggqs

File: src/components/Header/LayerRoute/LayerRoute.tsx (hunk 1 above)
Comment (enzoggqs) [reply src/components/Header/LayerRoute/LayerRoute.tsx]: Fixed!

File: src/components/Header/styles.ts (hunk 2 above)
Comment (enzoggqs) [reply src/components/Header/styles.ts:106]: I didnt understant this

Comment (LeoSilvaGomes) [reply src/components/Header/styles.ts:106]: It seen that you copy from here https://github.com/gpmc-lab-ufrgs/atlas/pull/41/files#:~:text=%7D-,export%20const%20ReturnRoute%20%3D%20styled.div%3CIReturnRoute%3E%60,-Write

File: src/components/Header/LayerRoute/LayerRoute.tsx (hunk 3 above)
Comment (enzoggqs) [reply src/components/Header/LayerRoute/LayerRoute.tsx:33]: oops, forgot about that component 😅

File: src/components/Header/LayerRoute/styles.ts [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  width: 20px;
+  height: 20px;
+  background-color: white;
+  margin-right: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:hover {
+    background-color: black;
```
Comment (LeoSilvaGomes) [inline src/components/Header/LayerRoute/styles.ts:53]: Could you upgrade this to #509e2f color, this color will fit better our context 👯

Comment (enzoggqs) [reply src/components/Header/LayerRoute/styles.ts:53]: of course!

Comment (enzoggqs) [reply src/components/Header/LayerRoute/styles.ts]: Done

**Comment by LeoSilvaGomes** [comment]
great great job

**Review by LeoSilvaGomes — APPROVED** [review]

File: src/components/Header/LayerRoute/styles.ts [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  .state {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'state' ? 'bold' : 'normal')};
+  }
+
+  .district {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'district' ? 'bold' : 'normal')};
+  }
+
+  Button {
+    padding: 5px;
```
Comment (LeoSilvaGomes) [inline src/components/Header/LayerRoute/styles.ts:32]:
```suggestion
    padding: 0px 5px;
```
for the button be more slim

**Review by valdersonjr — APPROVED** [review]
Looks good to me. Well done!

**Pull request merged**